### PR TITLE
Feature/logging

### PR DESCRIPTION
**Pull request by swaroopar**

### CODE REVIEW

File: modules/api/src/main/java/org/eclipse/osc/modules/api/OrchestratorApi.java [hunk 1]
```diff
@@ -31,21 +36,16 @@ public Response fetch(@HeaderParam("ocl") String oclLocation) throws Exception {
     @Path("/health")
     @GET
     @Produces(MediaType.TEXT_PLAIN)
-    public String health() throws Exception {
-        if (getOrchestrator() == null) {
-            throw new IllegalStateException("Orchestrator service is not ready");
-        }
+    public String health() {
```
Comment (jbonofre) [inline modules/api/src/main/java/org/eclipse/osc/modules/api/OrchestratorApi.java:39]: These changes are not related to logging right ?

File: plugins/huaweicloud/src/main/java/org/eclipse/osc/orchestrator/plugin/huaweicloud/HuaweiCloudOrchestratorPlugin.java [hunk 2]
```diff
@@ -61,7 +61,7 @@ public void startManagedService(String managedServiceName) {
             throw new IllegalArgumentException("Service:" + managedServiceName + "not registered.");
         }
 
-        Optional<AtomBuilder> optionalAtomBuilder = createBuiler(managedServiceName);
+        Optional<AtomBuilder> optionalAtomBuilder = createBuilder(managedServiceName);
```
Comment (jbonofre) [inline plugins/huaweicloud/src/main/java/org/eclipse/osc/orchestrator/plugin/huaweicloud/HuaweiCloudOrchestratorPlugin.java:64]: This typo fix is not related to logging. I have no problem to include in this PR, but the PR name could be "confusing".

File: pom.xml [hunk 3]
```diff
@@ -145,6 +145,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.10.1</version>
```
Comment (jbonofre) [inline pom.xml:148]: Same here: settings the plugin versions is not related to logging.

**Review by jbonofre — CHANGES_REQUESTED** [review]

**Comment by jbonofre** [comment]
I will create separated PRs for typo fix and maven plugin versions. I would focus on logging module in this PR (in a single commit), just to keep it simple.

File: modules/api/src/main/java/org/eclipse/osc/modules/api/OrchestratorApi.java (hunk 1 above)
Comment (swaroopar) [reply modules/api/src/main/java/org/eclipse/osc/modules/api/OrchestratorApi.java]: removed

**Comment by swaroopar** [comment]
Hi @jbonofre Thanks for your review. i have removed the commits not related to logging. But i'm not able to squash all into one commit since the parents have diverged for one commit. If allowed, we can try squashing them while merging to main. I will take care of this next time.

**Review by jbonofre — CHANGES_REQUESTED** [review]
Can you please squash all commits in one ?
To squash, you have to do:
```
git rebase -i origin/main
```
you keep the first commit as `pick` and you change all other commits as `squash` (instead of `pick`). Then, you will edit the commit message to something like `Adding OSC logging module`.
As you change the commit history, you have to force push on your branch:
```
git push --force
```
Then, you will have one commit instead of x ;)

**Comment by swaroopar** [comment]
@jbonofre Thanks for the hint. Commits are squashed now. Please review.

**Review by jbonofre — APPROVED** [review]
